.vscode/launch.json

```diff
@@ -0,0 +1,41 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```

Comment:
@anthonybernardi is this file supposed to be changed? im not sure if it is necessary.

Reply:
definitely not